src/components/ids-action-sheet/ids-action-sheet.ts

```diff
@@ -88,18 +88,22 @@ export default class IdsActionSheet extends Base {
    * Set the btn text attribute
    * @param {string} val the inner text of the cancel btn
    */
-  set btnText(val: string) {
+  set cancelBtnText(val: string) {
+    const idsButton = document.querySelector('ids-action-sheet')?.shadowRoot?.querySelector('ids-button') as HTMLElement;
```

Comment:
Each of the components should be "self-contained" as much as possible... For example, we already have a reference to "this" action sheet reference, so we want to use that as the starting point for reaching the button:

```suggestion
    const idsButton = this.shadowRoot?.querySelector('ids-button') as HTMLElement;
```